lib/src/package_analyzer.dart

```diff
@@ -46,12 +46,12 @@ class PackageAnalyzer {
       new PackageAnalyzer(await DartSdk.create(sdkDir: sdkDir),
           flutterDir: flutterDir, pubCacheDir: pubCacheDir);
 
-  Future<Summary> inspectPackage(
-    String package, {
-    String version,
-    bool keepTransitiveLibs: false,
-    Logger logger,
-  }) async {
+  Future<Summary> inspectPackage(String package,
+      {String version,
+      bool keepTransitiveLibs: false,
+      Logger logger,
+      bool deleteTemporaryDirectory: true}) async {
```

Comment:
I have a slight preference towards false default values, and in this case the name would be `keepTemporaryDirectory`. But the current works just as good.